cluster/charts/crossplane-types/templates/clusterrole.yaml

```diff
@@ -7,6 +7,25 @@ metadata:
     chart: {{ template "chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      rbac.crossplane.io/aggregate-to-crossplane-admin: "true"
+  - matchLabels:
+      rbac.crossplane.io/aggregate-to-crossplane: "true"
```

Comment:
Are you thinking that we could eventually have the stack manager add this label to newly installed roles, so we can differentiate between what `crossplane-admin` should grant and what `crossplane` should grant?

Reply:
Actually, I don't see a case where there is a ClusterRole that Crossplane creates with more permissions than Crossplane pod itself anymore (same treatment as stack-manager). So, the SA of Crossplane will have the most permissions regarding stacks and `crossplane-admin` would probably have the same level of permissions as it's _admin_. So I don't immediately see a case where we want to differentiate on stack level.

The differentiation exists in this file to some extent; aggregation `rbac.crossplane.io/aggregate-to-crossplane` is the minimal set that Crossplane needs without any stack. But as the stacks get installed, both `crossplane-admin` and `crossplane` end up having the same rules.